redisjson/src/main/java/site/ycsb/db/RedisJSONClient.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright (c) 2020 Redis Labs. All rights reserved.
```

Comment:
We generally try to keep everything in the project with copyright attributed to the YCSB project contributors. Is there some specific reason Redis Labs needs to maintain copyrights to this specific file?

Reply:
Full disclosure: I'm a Redis Labs employee. But the Apache license still applies.

Reply:
Sure. lots of folks contribute as a part of work product. Generally we still ask the employer to contribute the copyright as our group of contributors. Having individual copyrights listed per file makes it harder to maintain.

as an example, Google wanted their contributions called out so we added a blurb to the NOTICE file instead of having it in individual files. Can we do something like that instead?

Reply:
Copyright notice removed.